src/main/java/com/configcat/Segment.java

```diff
@@ -0,0 +1,29 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * ConfigCat segment.
+ */
+public class Segment {
+
+    /**
+     * The name of the segment.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.